.prettierignore

```diff
@@ -1,4 +1,8 @@
 **/*
 
 /.nx/cache
-/.nx/workspace-data
\ No newline at end of file
+/.nx/workspace-data
+
+!**/*.json
+**/package.json
```

Comment:
Why do we want to ignore package.json? Are there prettier rules we could use to standardize these?